gokart/run.py

```diff
@@ -86,19 +86,20 @@ def _try_to_send_event_summary_to_slack(slack_api: Optional[gokart.slack.SlackAP
     if slack_api is None:
         # do nothing
         return
+    options = gokart.slack.SlackConfig()
     with CmdlineParser.global_instance(cmdline_args) as cp:
         task = cp.get_task_obj()
-        tree_info = gokart.make_tree_info(task, details=True)
+        tree_info = gokart.make_tree_info(task, details=True) if options.send_tree_info else ''
```

Comment:
I suggest `else "Please add SlackConfig. send_tree_info to include tree-info" `, and do not delete Line 99 & 100 in master, and delete Line 101 & 102 in this branch.

Reply:
@nishiba Thanks for your point. This statement is replaced. And changed statement around Line 99 & 100 reverted.